### PR TITLE
Remove unnecessary attributes which are already filled from raw config

### PR DESCRIPTION
**Pull request by zilm13**
## PR Description
All these things are already added from rawConfig. We don't need to put it there once we add something to SpecConfig. So all things that already come from specConfig are removed.

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java [hunk 1]
```diff
@@ -36,25 +36,8 @@ public Map<String, String> getConfigMap() {
         .getRawConfig()
         .forEach((name, value) -> configAttributes.put(name, ConfigProvider.formatValue(value)));
 
-    configAttributes.put("GOSSIP_MAX_SIZE", getGossipMaxSize());
```
Comment (rolfyone) [inline data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java:39]: i'm concerned about removing these.

If they're in the YAML that we just got for holesky, why would we remove them? (only checked first one)

Comment (zilm13) [reply data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java:39]: all atributes already arrives from rawConfig line above except those from hardcoded constants

Comment (zilm13) [reply data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java:39]: we will not start with config without these attributes

**Review by rolfyone — APPROVED** [review]
LGTM, the main thing was to ensure they are still in the spec response on the api, as they'll be needed by anything getting the spec.

**Comment by tbenr** [comment]
> LGTM, the main thing was to ensure they are still in the spec response on the api, as they'll be needed by anything getting the spec.

So is there any possibility that one or more of this params do not come from `getRawConfig`?

**Comment by rolfyone** [comment]
So when network=auto on the VC and it hits the spec endpoint for example...

**Comment by zilm13** [comment]
Here is the spec answer in this branch running locally:
```
{
  "data": {
    "SLOTS_PER_EPOCH": "32",
    "PRESET_BASE": "mainnet",
    "TERMINAL_TOTAL_DIFFICULTY": "10790000",
    "INACTIVITY_SCORE_BIAS": "4",
    "MAX_ATTESTER_SLASHINGS": "2",
    "MAX_WITHDRAWALS_PER_PAYLOAD": "16",
    "INACTIVITY_PENALTY_QUOTIENT_BELLATRIX": "16777216",
    "INACTIVITY_PENALTY_QUOTIENT": "67108864",
    "SAFE_SLOTS_TO_UPDATE_JUSTIFIED": "8",
    "SECONDS_PER_ETH1_BLOCK": "14",
    "MIN_SEED_LOOKAHEAD": "1",
    "VALIDATOR_REGISTRY_LIMIT": "1099511627776",
    "SLOTS_PER_HISTORICAL_ROOT": "8192",
    "RESP_TIMEOUT": "10",
    "DOMAIN_VOLUNTARY_EXIT": "0x04000000",
    "MAX_VALIDATORS_PER_COMMITTEE": "2048",
    "MIN_GENESIS_TIME": "1614588812",
    "ALTAIR_FORK_EPOCH": "36660",
    "HYSTERESIS_QUOTIENT": "4",
    "ALTAIR_FORK_VERSION": "0x01001020",
    "MAX_BYTES_PER_TRANSACTION": "1073741824",
    "MAX_CHUNK_SIZE": "10485760",
    "TTFB_TIMEOUT": "5",
    "WHISTLEBLOWER_REWARD_QUOTIENT": "512",
    "PROPOSER_REWARD_QUOTIENT": "8",
    "MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP": "16384",
    "EPOCHS_PER_HISTORICAL_VECTOR": "65536",
    "MIN_PER_EPOCH_CHURN_LIMIT": "4",
    "TARGET_AGGREGATORS_PER_SYNC_SUBCOMMITTEE": "16",
    "MAX_DEPOSITS": "16",
    "BELLATRIX_FORK_EPOCH": "112260",
    "TARGET_AGGREGATORS_PER_COMMITTEE": "16",
    "DOMAIN_SYNC_COMMITTEE_SELECTION_PROOF": "0x08000000",
    "MESSAGE_DOMAIN_INVALID_SNAPPY": "0x00000000",
    "EPOCHS_PER_SLASHINGS_VECTOR": "8192",
    "MIN_SLASHING_PENALTY_QUOTIENT": "128",
    "MAX_BLS_TO_EXECUTION_CHANGES": "16",
    "GOSSIP_MAX_SIZE": "10485760",
    "DOMAIN_BEACON_ATTESTER": "0x01000000",
    "EPOCHS_PER_SUBNET_SUBSCRIPTION": "256",
    "ATTESTATION_SUBNET_COUNT": "64",
    "GENESIS_DELAY": "1919188",
    "MAX_SEED_LOOKAHEAD": "4",
    "ETH1_FOLLOW_DISTANCE": "2048",
    "SECONDS_PER_SLOT": "12",
    "MIN_SYNC_COMMITTEE_PARTICIPANTS": "1",
    "BELLATRIX_FORK_VERSION": "0x02001020",
    "PROPORTIONAL_SLASHING_MULTIPLIER_BELLATRIX": "3",
    "EFFECTIVE_BALANCE_INCREMENT": "1000000000",
    "FIELD_ELEMENTS_PER_BLOB": "4096",
    "MIN_EPOCHS_TO_INACTIVITY_PENALTY": "4",
    "BASE_REWARD_FACTOR": "64",
    "MAX_EXTRA_DATA_BYTES": "32",
    "CONFIG_NAME": "prater",
    "MAX_PROPOSER_SLASHINGS": "16",
    "INACTIVITY_SCORE_RECOVERY_RATE": "16",
    "MAX_TRANSACTIONS_PER_PAYLOAD": "1048576",
    "DEPOSIT_CONTRACT_ADDRESS": "0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b",
    "MIN_ATTESTATION_INCLUSION_DELAY": "1",
    "SHUFFLE_ROUND_COUNT": "90",
    "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH": "18446744073709551615",
    "MAX_EFFECTIVE_BALANCE": "32000000000",
    "DOMAIN_BEACON_PROPOSER": "0x00000000",
    "DOMAIN_SYNC_COMMITTEE": "0x07000000",
    "PROPOSER_SCORE_BOOST": "70",
    "DOMAIN_SELECTION_PROOF": "0x05000000",
    "MIN_SLASHING_PENALTY_QUOTIENT_BELLATRIX": "32",
    "HYSTERESIS_UPWARD_MULTIPLIER": "5",
    "SUBNETS_PER_NODE": "2",
    "MIN_DEPOSIT_AMOUNT": "1000000000",
    "PROPORTIONAL_SLASHING_MULTIPLIER_ALTAIR": "2",
    "MAX_BLOBS_PER_BLOCK": "6",
    "MIN_VALIDATOR_WITHDRAWABILITY_DELAY": "256",
    "MAXIMUM_GOSSIP_CLOCK_DISPARITY": "500",
    "TARGET_COMMITTEE_SIZE": "128",
    "TERMINAL_BLOCK_HASH": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "DOMAIN_DEPOSIT": "0x03000000",
    "DOMAIN_CONTRIBUTION_AND_PROOF": "0x09000000",
    "UPDATE_TIMEOUT": "8192",
    "SYNC_COMMITTEE_BRANCH_LENGTH": "5",
    "DEPOSIT_CHAIN_ID": "5",
    "MAX_BLOB_COMMITMENTS_PER_BLOCK": "4096",
    "DOMAIN_RANDAO": "0x02000000",
    "CAPELLA_FORK_VERSION": "0x03001020",
    "MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR": "64",
    "EPOCHS_PER_ETH1_VOTING_PERIOD": "64",
    "HISTORICAL_ROOTS_LIMIT": "16777216",
    "ATTESTATION_PROPAGATION_SLOT_RANGE": "32",
    "SYNC_COMMITTEE_SIZE": "512",
    "ATTESTATION_SUBNET_PREFIX_BITS": "6",
    "PROPORTIONAL_SLASHING_MULTIPLIER": "1",
    "MESSAGE_DOMAIN_VALID_SNAPPY": "0x01000000",
    "MAX_VOLUNTARY_EXITS": "16",
    "HYSTERESIS_DOWNWARD_MULTIPLIER": "1",
    "DOMAIN_APPLICATION_BUILDER": "0x00000001",
    "EPOCHS_PER_SYNC_COMMITTEE_PERIOD": "256",
    "BYTES_PER_LOGS_BLOOM": "256",
    "MIN_GENESIS_ACTIVE_VALIDATOR_COUNT": "16384",
    "MAX_ATTESTATIONS": "128",
    "MIN_EPOCHS_FOR_BLOCK_REQUESTS": "33024",
    "MAX_REQUEST_BLOCKS": "1024",
    "GENESIS_FORK_VERSION": "0x00001020",
    "DEPOSIT_NETWORK_ID": "5",
    "SYNC_COMMITTEE_SUBNET_COUNT": "4",
    "CAPELLA_FORK_EPOCH": "162304",
    "EJECTION_BALANCE": "16000000000",
    "ATTESTATION_SUBNET_EXTRA_BITS": "0",
    "MAX_COMMITTEES_PER_SLOT": "64",
    "SHARD_COMMITTEE_PERIOD": "256",
    "INACTIVITY_PENALTY_QUOTIENT_ALTAIR": "50331648",
    "DOMAIN_AGGREGATE_AND_PROOF": "0x06000000",
    "CHURN_LIMIT_QUOTIENT": "65536",
    "BLS_WITHDRAWAL_PREFIX": "0x00"
  }
}
```
So everything "removed" is included:
```
    "RESP_TIMEOUT": "10",
    "MAX_CHUNK_SIZE": "10485760",
    "TTFB_TIMEOUT": "5",
    "MESSAGE_DOMAIN_INVALID_SNAPPY": "0x00000000",
    "GOSSIP_MAX_SIZE": "10485760",
    "EPOCHS_PER_SUBNET_SUBSCRIPTION": "256",
    "ATTESTATION_SUBNET_COUNT": "64",
    "SUBNETS_PER_NODE": "2",
    "MAXIMUM_GOSSIP_CLOCK_DISPARITY": "500",
    "ATTESTATION_PROPAGATION_SLOT_RANGE": "32",
    "ATTESTATION_SUBNET_PREFIX_BITS": "6",
    "MESSAGE_DOMAIN_VALID_SNAPPY": "0x01000000",
    "MIN_EPOCHS_FOR_BLOCK_REQUESTS": "33024",
    "MAX_REQUEST_BLOCKS": "1024",
    "ATTESTATION_SUBNET_EXTRA_BITS": "0",
```

**Review by tbenr — APPROVED** [review]
LGTM

**Pull request merged**